interface/src/app/node/high_range_exp.tpl.html

```diff
@@ -0,0 +1,48 @@
+<div ng-if="!queryStatus">
+  <h4>
+    <b>Experiments with the highest range: </b>{{experiments.length}}
+  </h4>
+  <button class="btn btn-primary" ng-click="showTop()"
+          ng-hide="topMode">Show Top {{topNum}}</button>
+  <button class="btn btn-primary" ng-click="showAll()"
+          ng-show="topMode">Show All</button>
+
+  <br />
+  (Click
+  <i class="fa fa-plus-square fa-lg" aria-hidden="true"></i> or
```

Comment:
As I was playing around with the interface, I was wondering if these icons (on the table below) should be actual buttons so that users intuitively know that they can click on them.